qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -139,3 +141,58 @@ def status(self):
             raise LookupError(
                 "Couldn't get backend status: {0}".format(ex))
         return status
+
+    def jobs(self, limit=50, skip=0, status=None):
+        """Attempt to get the jobs submitted to the backend
+
+        Args:
+            limit (int): number of jobs to retrieve
+            skip (int): starting index of retrieval
+            status (None or JobStatus or str): only get jobs with this status,
+                where status is e.g. JobStatus.RUNNING or 'RUNNING'.
+        Returns:
+            list(IBMQJob): list of IBMQJob instances
+        """
+        backend_name = self.configuration['name']
+        job_list = []
+        base_index = skip
+        job_info_list = []
+        if isinstance(status, str):
+            status = JobStatus[status]
+        while len(job_list) < limit or len(job_info_list) < limit:
+            job_info_list = self._api.get_jobs(limit=limit, skip=base_index,
```

Comment:
With the latest version of IBMQuantumExperience package (1.9.4), we can pass a `filter` to the [get_jobs()](https://github.com/QISKit/qiskit-api-py/blob/580a57c8fdbadcd4041843d29946753021002181/IBMQuantumExperience/IBMQuantumExperience.py#L777), so we can filter by backend name and job status, making this whole algorithm way simpler and reducing the potential huge number of requests to 1.

Reply:
That's great! Thanks for getting that through.